documentation/docs/steps/transportRequestUploadRFC.md

```diff
@@ -0,0 +1,101 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have enabled RFC on the ABAP system.
+* You have an user account on the ABAP system and the roles required for uploading via RFC.
+* You have created a transport request on the ABAP system, which is the target container of the upload.
+* You have installed an RFC library based Connector Client. See [RFC Client](#[RFC Client).
+
+## RFC Client
+
+Create a Docker image as described in the Git repository [devops-docker-images/node-rfc](https://github.com/rodibrin/devops-docker-images/tree/master/node-rfc).
+Put the image in its own Docker register in [Docker Hub](https://hub.docker.com/).
+Make the image known to the pipeline in the configuration.
+
+```yaml
+steps:
+  transportRequestUploadRFC:
+    dockerImage: 'my/rfc-client'
+```
+
+The RFC Client connects to the ABAP system using the [SAP NetWeaver RFC SDK](https://support.sap.com/en/product/connectors/nwrfcsdk.html).
```

Comment:
- I suggest "your ABAP system*" 
- I would add a black line afterwards so that line 24 and 25 don't appear in one line.